src/source/Rtcp/RtcpPacket.c

```diff
@@ -122,7 +122,7 @@ STATUS rembValueGet(PBYTE pPayload, UINT32 payloadLen, PDOUBLE pMaximumBitRate,
     mantissa &= RTCP_PACKET_REMB_MANTISSA_BITMASK;
 
     exponent = pPayload[RTCP_PACKET_REMB_IDENTIFIER_OFFSET + SIZEOF(UINT32) + SIZEOF(BYTE)] >> 2;
-    maximumBitRate = ldexp(mantissa, exponent);
+    maximumBitRate = mantissa * (1 << exponent);
```

Comment:
Minor comment, this could also be: 
`maximumBitRate = mantissa << exponent`

Reply:
That won’t be equivalent right? Here, we are trying to multiply mantissa with the 2 in the power of the exponent. Doesn’t it?